python/seldon_core/env_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
Could we just have these in the `utils.py`? Is there a particular reason to have a separate utils file/